zanata-war/src/main/java/org/zanata/servlet/UrlRewriteConfig.java

```diff
@@ -66,7 +66,7 @@ public Configuration getConfiguration(final ServletContext context) {
 
                 .addRule(Join.path("/error/").to("/error.xhtml"))
                 .addRule(Join.pathNonBinding("/error/{path}").to("/error/{path}.xhtml"))
-                .addRule(Join.path("/warn").to("/warn.xhtml"))
+//                .addRule(Join.path("/error/missing_entity").to("/error/missing_entity.xhtml"))
```

Comment:
- [ ] If the missing entity error page is working without this line, we shouldn't need the comment.